shoop/admin/templates/shoop/admin/system/telemetry.jinja

```diff
@@ -0,0 +1,56 @@
+{% extends "shoop/admin/base.jinja" %}
+{% block content %}
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-md-4">
+                <h2>About Shoop Telemetry</h2>
+                <p>Shoop will occasionally send telemetry data to a Shoop.io server.</p>
+                <p>
+                    The data contains an unique installation key,
+                    as well as the hostname of the installation (as sent by the HTTP client).
```

Comment:
I don't understand "HTTP client" in this context and I bet many Shoop administrator will feel the same.